packages/contracts-bedrock/contracts/test/Hashing.t.sol

```diff
@@ -21,19 +21,20 @@ contract Hashing_Test is CommonTest {
     }
 
     function test_hashCrossDomainMessage_differential(
-        uint256 _nonce,
+        uint240 _nonce,
+        uint16 _version,
         address _sender,
         address _target,
         uint256 _value,
         uint256 _gasLimit,
         bytes memory _data
     ) external {
-        // Discard any fuzz tests with an invalid version
-        (, uint16 version) = Encoding.decodeVersionedNonce(_nonce);
-        vm.assume(version < 2);
+        // Ensure the version is valid
+        uint16 version = uint16(bound(uint256(_version), 0, 1));
```

Comment:
I'd like to double check here that the final arg is inclusive

Reply:
Yes:
https://github.com/foundry-rs/forge-std/blob/33d4895fb629aabf12e0f60740367bd03e702bd1/src/StdUtils.sol#L15

I also [instrumented the code](https://github.com/maurelian/optimism-1/compare/develop...maurelian:optimism-1:vm-bound-counts?expand=1#diff-2e77d00d31481bd3b8012d5c4c74bb334d95ec7c74485825aa437af34357c53cR35-R38) to see the version values that were used, you [can see](https://github.com/maurelian/optimism-1/blob/b5eab76b34cefe19ba39bd2b125d1d057a7b4bb7/packages/contracts-bedrock/runs.txt) that it includes both 0s and 1s.